src/simplewallet/simplewallet.cpp

```diff
@@ -3266,6 +3323,7 @@ bool simple_wallet::init(const boost::program_options::variables_map& vm)
 
   m_wallet->callback(this);
 
+  m_electrum_seed.wipe();
```

Comment:
Would it also make sense to do `m_recovery_key.srub();` here? Or perhaps it should be done with `create_scope_leave_handler` to be sure.

Reply:
Seems like it would be a good idea, yes.

Reply:
I moved the seed wiping to a leave handler. I'm not adding the recovery key thing, it'll get done when I get to wiping more stuff later.

Reply:
Ok